events/app/views/pages/events.njk

```diff
@@ -0,0 +1,34 @@
+{% extends 'layout/default.njk' %}
+
+{% block pageMeta %}
+  {% set metaContent = {
+    type: 'website',
+    title: 'Events',
+    image: allEvents[0].image.contentUrl,
+    url: pageConfig.canonicalUri,
+    description: 'Past and present events at Wellcome Collection'
```

Comment:
Let's try and get this written - not blocking, but good if we could.
@jennpb?

Reply:
Or can we put something really crap in here e.g. INSERT DESCRIPTION HERE, and address it when we can.